src/main/java/mods/eln/fluid/FuelRegistry.kt

```diff
@@ -27,6 +27,7 @@ object FuelRegistry {
             "rc ethanol" to 21172000.0, // RotaryCraft, density = 0,79 kg/l, heating value = 26.8 MJ/kg
             "biofuel" to 17826480.0,    // Minefactory Reloaded, (Bioethanol) density = 0.786 kg/l, heating value = 22,68 MJ/l
             "bioethanol" to 17826480.0, // Forestry
+            "gasoline" to 25820000.0,   // PneumaticCraft, density = 0.755 kg/L, heat value = 34,2 MJ/l
```

Comment:
How could be gasoline not on the gasoline list? ^^

Should fuel and gasoline not have about the same heating value?